### PR TITLE
fix: peers format

### PR DESCRIPTION
**Pull request by hugoArregui**

### CODE REVIEW

File: src/ports/comms-stats.ts [hunk 1]
```diff
@@ -61,6 +61,7 @@ export async function createCommsStatsComponent(
     const islandsSubscription = nats.subscribe('archipelago.islands')
     ;(async () => {
       for await (const message of islandsSubscription.generator) {
+        logger.info('got islands message')
```
Comment (agusaldasoro) [inline src/ports/comms-stats.ts:64]: is this log ok?

File: src/controllers/handlers/peers-handler.ts [hunk 2]
```diff
@@ -27,7 +23,7 @@ export async function peersHandler(
     components: { commsStats }
   } = context
 
-  const peers = await commsStats.getPeers()
+  const peers = commsStats.getPeers()
```
Comment (agusaldasoro) [inline src/controllers/handlers/peers-handler.ts:26]: why removing the await?

**Review by agusaldasoro — APPROVED** [review]

File: src/controllers/handlers/peers-handler.ts (hunk 2 above)
Comment (hugoArregui) [reply src/controllers/handlers/peers-handler.ts:26]: doesn't return a promise

File: src/ports/comms-stats.ts (hunk 1 above)
Comment (hugoArregui) [reply src/ports/comms-stats.ts]: removed

**Pull request merged**